package.json

```diff
@@ -30,8 +30,11 @@
   "steal": {
```

Comment:
Maybe this should be `system` to remain compat with steal 0.16? But you didn't change this so don't want to block the PR on account of this.